shadow-dom/focus/text-selection-with-delegatesFocus.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<body>
+    <x-shadow id="withoutFocus"></x-shadow>
+    <x-shadow id="withFocus"></x-shadow>
+</body>
+
+<script>
+'use strict';
+
+/**
+ * build shadow-root with delegates focus, a focusable element, and an (ideally) selectable text
+ */
+function buildShadowRootWithSelectableText( element, shouldDelegateFocus ) {
+    element.attachShadow({ mode: 'open', delegatesFocus: shouldDelegateFocus });
+    const span = document.createElement('span');
+    span.textContent = 'Example Text to Select ';
+    const button = document.createElement('button');
+    button.textContent = 'Button';
+    element.shadowRoot.append(span);
```

Comment:
nit: you can do `.append(span,button)`

Reply:
ah, that didn't even occur to me! Will update!